tests/Zend/Controller/Action/HelperBroker/PriorityStackTest.php

```diff
@@ -57,10 +57,10 @@ public function testStackMaintainsLifo()
         $this->stack->push(new Zend_Controller_Action_Helper_ViewRenderer());
         $this->stack->push(new Zend_Controller_Action_Helper_Redirector());
         $this->assertEquals(2, count($this->stack));
-        $iterator = $this->stack->getIterator();
-        $this->assertEquals('Zend_Controller_Action_Helper_Redirector', get_class(current($iterator)));
-        next($iterator);
-        $this->assertEquals('Zend_Controller_Action_Helper_ViewRenderer', get_class(current($iterator)));
+        $iterator = $this->stack->getIterator()->getIterator();
+        $this->assertEquals('Zend_Controller_Action_Helper_Redirector', get_class($iterator->current()));
+        $iterator->next();
+        $this->assertEquals('Zend_Controller_Action_Helper_ViewRenderer', get_class($iterator->current()));
```

Comment:
does php8 throw on calling `current()` / `next()` on an iterator?

Reply:
It retuns false and thus get_class throws a TypeError